README.md

```diff
@@ -48,7 +48,7 @@ Receive notifications of critical security updates. The sophistication of online
 
 - [Contribute Code](https://developers.mattermost.com/contribute/getting-started/)
 - [Find "Help Wanted" projects](https://github.com/mattermost/mattermost-server/issues?page=1&q=is%3Aissue+is%3Aopen+%22Help+Wanted%22&utf8=%E2%9C%93)
-- [Join Developer Discussion on a Mattermost Server for contributors](https://pre-release.mattermost.com/signup_user_complete/?id=f1924a8db44ff3bb41c96424cdc20676)
+- [Join Developer Discussion on a Mattermost Server for contributors](https://community.mattermost.com/signup_user_complete/?id=f1924a8db44ff3bb41c96424cdc20676)
```

Comment:
The other occurrences link to the server blank, this one linkts to the `/signup_user_complete/?id=...` part. Is the id relevant to track sign-ups from the GitHub repo or just an artifact? 
In case it's meant for tracking it should probably be put into every occurence, in the other case I think it should probably be discarded. What do you think?

Reply:
@lenucksi the ID ensures that they get automatically invited to the Contributors team when they sign up, and don't have to explicitly join a team. Saves new community members a step. Not used for tracking

But agree, we can use the same link for both occurrences

Reply:
Thanks for the insight!